app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+As a music teacher you'll inspire pupils to develop a love of music and their talent as musicians. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/music-secondary-ks3/units">explore what teaching a music lesson could look like</a>.
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=W3&visa_status=false">Search for music teacher training courses</a>.
+
+## Philosophy 
+
+As a philosophy teacher you'll teach pupils ways of thinking about ethics, time and meaning. Help them explore the big questions of life and morality. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=P1&visa_status=false">Search for philosophy teacher training courses</a>.
+
+## Physical education 
+
+As a P.E teacher you'll help pupils to develop motor skills and an interest for physical activity and fitness. 
+
+You can <a href="https://www.thenational.academy/teachers/programmes/physical-education-secondary-ks3-l/units">explore what teaching a P.E lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C6&visa_status=false">Search for physical education teacher training courses</a>.
+
+
+## Physical education with an EBacc subject
+
+You'll be able to teach P.E teacher and another subject from: English, maths, history, languages or geography.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=C7&visa_status=false">Search for physical education with an EBacc subject teacher training courses</a>.
+
+## Physics
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=physics&button=#funding-widget">Tax-free scholarships of £30,000 and bursaries of £28,000 are available for eligible trainee physics teachers</a>. 
+
+<a href="/is-teaching-right-for-me/physics">You can learn more about teaching physics</a>. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=F3&visa_status=false">Search for physics teacher training courses</a>.
+
+## Psychology 
+
+As a psychology teacher you'll help pupils develop an understanding of the ideas and values that characterise ‘self’ and others. 
```

Comment:
@caitlin-reidy I don't really know what this sentence means? Can we say it in a more conversational way? What sorts of jobs can pupils who study psychology go on to do? Can we give any relatable examples of what they'll teach?